liquibase-standard/src/test/groovy/liquibase/changelog/ChangeLogParametersTest.groovy

```diff
@@ -145,4 +145,17 @@ class ChangeLogParametersTest extends Specification {
         then:
         expanded == "1234"
     }
+
+    def "System properties set up naively are not filtered when using required labels or contexts"() {
+        when:
+        def database = new MockDatabase()
+
+        System.setProperty("SOME_PROPERTY", "This should not be filtered")
```

Comment:
You should unset this System property in a cleanup block in this test.

Reply:
Done!